snap-style.gemspec

```diff
@@ -20,7 +20,8 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
 
-  spec.add_dependency "rubocop", "~> 0.63.0"
+  spec.add_dependency "rubocop", "~> 0.81.0"
+  spec.add_dependency "rubocop-rails", "~> 2.5.2"
```

Comment:
We should leave this for now, but I question if it's actually used.
I know we "enable rails", but I don't think any of these cops are for Rails. Might be better to just let the consumer add that when necessary.

Reply:
Yeah, I'm not really sure. I *think* you're right that none of the cops are for Rails.

Reply:
Looks like VBE disables the Rails cop anyhow.